docker/dotnet.dockerfile

```diff
@@ -1,4 +1,19 @@
 FROM mcr.microsoft.com/dotnet/core/sdk:2.1
 
+RUN apt-get update
```

Comment:
Should we have separate dockerfiles instead? Another one with `mcr.microsoft.com/dotnet/core/sdk:3.0`?

Reply:
Yeah I think it would be best to have separate dockerfiles, but I think that might require a good deal of re-architecting our docker compose files. Would you mind if this work was delayed to another PR?

Reply:
Definitely!